### DISCUSSION TITLE
EE6 - Accessibility issues

### DISCUSSION THREAD

**Discussion by blue-dreamer** [ExpressionEngine/ExpressionEngine#2915] [Feature Requests]
**Description of the problem**

Several text elements don't meet level AA, for instance inactive tab text, field instructions and grid column headings on the publish page. 

Many form fields don't have associated labels, on publish page that seems to be for most fields.

Fieldsets are missing LEGEND tags

**Possible Solution**
Tweak so the CP is at least level AA.

**Additional context**

Some organisations look at accessibility when choosing a CMS, and many have disabled staff who use it. Having something that meets a good standard is a selling point!

**Comment by ops-andy** [comment]
@blue-dreamer thanks for pointing this out. While we've announced that we want to move toward better accessibility, we know we still have a long way to go and these kinds of items are definitely already on the list.

**Comment by blue-dreamer** [comment]
Good to know thanks!

**Comment by Yulyaswan** [comment]
improving accessibility in this PR - #3511
